dask_sql/server/responses.py

```diff
@@ -87,7 +88,9 @@ def get_data_description(df):
     @staticmethod
     def convert_cell(cell):
         try:
-            if np.isnan(cell):  # pragma: no cover
+            if pd.isna(cell):
```

Comment:
I'm not completely familiar with the server side of dask-sql but is `cell` always expected to be a `pandas` / host based object?

Reply:
Yeah, at this point in the response generation we would've called `to_pandas` on any cuDF frames